test/smoke/testApps/SpringBootTest/src/main/java/com/springbootstartertest/controller/TargetObject.java

```diff
@@ -0,0 +1,26 @@
+package com.springbootstartertest.controller;
+
+class TargetObject {
+
+    String one() {
+        return "One";
+    }
+
+    String two() {
+        return "Two";
+    }
+
+    String two(String input) {
+        return input;
+    }
+
+    String three() throws Exception {
+        throw new Exception("Three");
+    }
+
+    static class NestedObject {
+
+        void four(boolean x, int[] y, String[][] z) {
+        }
+    }
+}
```

Comment:
Could we add a case like this:
```java
String five() {
    StringBuilder sb = new StringBuilder();
    sb.append(one());
    sb.append(two("call_two"));
    four(false, new int[0], new String[0][0]);
    return sb.append("_Five").toString();
}
```
I've seen this pattern before where a service is monitoring the high-level operations as well as the low-level parts of those operations.

Reply:
Done, and this uncovered a bug that prior tests missed!